src/main/java/io/stargate/sgv2/jsonapi/service/operation/model/impl/CreateCollectionOperation.java

```diff
@@ -15,17 +15,23 @@ public record CreateCollectionOperation(
     String name,
     boolean vectorSearch,
     int vectorSize,
-    String vectorFunction)
+    String vectorFunction,
+    String vectorize)
```

Comment:
Is this a JSON serialization of Vectorize service configuration?

Reply:
Yes, the value is stored part of table's comment.